codice/ddf
# DDF-2298: Registry name field now auto populates

#### What does this PR do?

When connecting to a remote registry, the name field has been removed. A placeholder name is now determined by the hostname and port number of the service url until a connection to the remote registry can be made. Once the connection has been made, the name field is assigned the name of the remote registry.
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?

@clockard @gordocanchola @vinamartin @mcalcote
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@pklinef 
@shaundmorris 
#### How should this be tested?

You will need 2 DDF's with security or use loopback.
Setup a remote registry using the url of the other ddf. The name will populate with the hostname:port of the remote registry and connection type.
Once the other DDF becomes available, the name field will change to be the remote registries name given by the admin + hostname:port + connection type.
Make sure that no duplicate values are entered and that the UI correctly allows you to edit the url information. Check that the UI disallows urls of the same connection type that are also on the same hostname:port (ex: the UI does not allow two csw connections over the same hostname:port).
#### Any background context you want to provide?

This PR was mistakenly closed. Please see https://github.com/codice/ddf/pull/997 for previous conversations on the PR.
#### What are the relevant tickets?

DDF-2298
#### Screenshots (if appropriate)
#### Checklist:

When a user subscribes to a remote registry, the name is determined before connecting
using the hostname and port number of the given service url. Upon successful
connection to the remote registry, the remote name assigned to the registry is used
as part of it's name.
